### ISSUE TITLE
put version number into error messages

### ISSUE THREAD

**Issue by jeremybmerrill** [tabulapdf/tabula#297]
so when people copy/paste us the error message, we know what version they were using

also: this will require a way to get the version number into JS land (probably via a fake silly endpoint for now, maybe something better later)

**Comment by jazzido** [comment]
Some time ago I added code to the build scripts to bake the version number into some of the templates. That was used to show that silly "DEV" ribbon if you were running from a git checkout.

**Comment by jeremybmerrill** [comment]
Yes, I've seen it, but it depends on rendering erb templates, which I'm trying to get rid of entirely. Not opposed to keeping that for now, but might be easier to make an itty-bitty API endpoint -- so we can further decouple the UI from the API (in order to eventually make them separate repos).

**Comment by jeremybmerrill** [comment]
fixed in a01f14f; error message text now looks like:
```
Tabula API version: rev24705ad59749a47b083dd76e0dcb3b9a2b8fe7d8
Filename: cscity.pdf
IOError: IOError
	/home/merrillj/code/tabula/webapp/tabula_web.rb:263:in `(root)'
	/home/merrillj/.rbenv/versions/jruby-1.7.15/lib/ruby/gems/shared/gems/cuba-3.3.0/lib/cuba.rb:0:in `on'
	/home/merrillj/.rbenv/versions/jruby-1.7.15/lib/ruby/gems/shared/gems/cuba-3.3.0/lib/cuba.rb:200:in `try'
	/home/merrillj/.rbenv/versions/jruby-1.7.15/lib/ruby/gems/shared/gems/cuba-3.3.0/lib/cuba.rb:161:in `on'
	/home/merrillj/code/tabula/webapp/tabula_web.rb:262:in `(root)'
	/home/merrillj/.rbenv/versions/jruby-1.7.15/lib/ruby/gems/shared/gems/cuba-3.3.0/lib/cuba.rb:0:in `on'
```
